libs/unity/library/Runtime/Scripts/NativeRender/NativeRenderer.cs

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+
+namespace Microsoft.MixedReality.WebRTC.UnityPlugin
```

Comment:
This is too similar with `Microsoft.MixedReality.WebRTC.Unity` and the difference is not clear. It should either be merged with `Microsoft.MixedReality.WebRTC.Unity` or it could be a sub-namespace, e.g. `Microsoft.MixedReality.WebRTC.Unity.Renderer`.

Reply:
Changed all UnityPlugin namespaces to just Unity.